board/common/overlay/sbin/fwupdate

```diff
@@ -37,11 +41,6 @@ function exit_usage() {
     exit 1
 }
 
-if [[ -z "$1" ]]; then
```

Comment:
Why is this check gone? Do we ever want to run `fwupdate` without a subcommand and not show the usage?

Reply:
It's not gone, it's just executed further down, when variables are available. These variables are required in exit_usage to decide whether the `fwupdate list` feature will be shown.

Reply:
Oh, I somehow missed it.